app/routes.py

```diff
@@ -1,2 +1,59 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
+
+
+class Planet:
+    def __init__(self, name, id, description, color):
+        self.name = name
+        self.id = id
+        self.description = description
+        self.color = color
+    
+    def to_json(self):
+        return {"Name": self.name,
+                "ID": self.id,
+                "Description": self.description,
+                "Color": self.color
+                }
+
+planets = [
+    Planet("Earth", 3, "Earth is the 3rd planet in solar system", "blue"),
+    Planet("Mars", 4, "Mars is the 4th planet in solar system", "red"),
+    Planet("Jupiter", 5, "Jupiter is the 5th planet in solar system", "tan")
+]
+
+def validate_planet(id):
+    try:
+        id = int(id)
+    except:
+        return abort(make_response({"message": f"planet {id} is invalid"}, 400))
+    for planet in planets :
+        if planet.id == id:
+            return jsonify(planet.to_json()), 200
+    return abort(make_response({"message": f"planet {id} is not found"}, 404))
+# first string is flask blueprint name
+# run in localhost need to add/planets at the end
+solar_bp = Blueprint("", __name__, url_prefix="/planets")
+
+
+@solar_bp.route("", methods=["GET"])
+def see_planets():
```

Comment:
There's no hard and fast rule about this, but usually for these methods we can use words like 'read' or 'get'. So you could call this method "read_all" or "get_all".

In the future, if you end up splitting your routes up into distinct route files (planet_routes.py and moon_routes.py for example in a routes directory) then you could have method names like get_all() and get_one() without adding the word planet in the function name since all the routes will be related to planet.

Reply:
makes sense!